concent-builder/files/jenkins/Jenkinsfile

```diff
@@ -0,0 +1,76 @@
+properties([[$class:
+'GithubProjectProperty',
+    displayName: '',
+    projectUrlStr: 'https://github.com/bartoszbetka/concent-deployment.git/'],
```

Comment:
This is wrapped in a weird way. Hard to see what is nested in what. Why not do this?
``` groovy
properties([
    [
        $class:        'GithubProjectProperty',
        displayName:   '',
        projectUrlStr: 'https://github.com/bartoszbetka/concent-deployment.git/'
    ],
    pipelineTriggers([githubPush()])
])
```